tests/test_interfaces.py

```diff
@@ -474,19 +474,15 @@ def test_optimize(model_fixture, start_time, end_time, num_samples):
         assert bounds_interventions[0][i] <= opt_policy[i]
         assert opt_policy[i] <= bounds_interventions[1][i]
 
-    intervention_time = list(optimize_kwargs["static_parameter_interventions"].keys())
-    intervened_params = optimize_kwargs["static_parameter_interventions"][
-        intervention_time[0]
-    ]
     result_opt = sample(
         model_url,
         end_time,
         logging_step_size,
         num_samples,
         start_time=start_time,
-        static_parameter_interventions={
```

Comment:
Could you replace this `static_parameter_interventions=...` line with just `**optimize_kwargs`?

Reply:
Fixed it to use the optimal policy